packages/features/webhooks/lib/service/WebhookService.ts

```diff
@@ -0,0 +1,294 @@
+import { createHmac } from "crypto";
+
+import logger from "@calcom/lib/logger";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+import type { WebhookSubscriber, WebhookDeliveryResult } from "../dto/types";
+import type { WebhookPayload } from "../factory/types";
+import type { ITasker } from "../interface/infrastructure";
+import type { IWebhookRepository, IWebhookService } from "../interface/services";
+import { TaskerProvider } from "../provider/TaskerProvider";
+
+const log = logger.getSubLogger({ prefix: ["[WebhookService]"] });
```

Comment:
Also, I think we started injecting _log_ as well as a dependency. So, no import of logger as well in any module.